README.md

```diff
@@ -1,34 +1,38 @@
-# Cloud Native Meetup Linz Website
+# Cloud Native Linz Website
 
 ![GitHub last commit](https://img.shields.io/github/last-commit/cloudnativelinz/cloudnativelinz.github.io?label=updated)
 ![Website](https://img.shields.io/website?url=https%3A%2F%2Fcloudnativelinz.github.io)![GitHub contributors](https://img.shields.io/github/contributors/cloudnativelinz/cloudnativelinz.github.io)
 [![Twitter Follow](https://img.shields.io/twitter/follow/cloudnativelinz?style=social)](https://twitter.com/cloudnativelinz)
 
 ## Visit the website
 
-Find the website here: https://cloudnativelinz.github.io.
+Find the website here: <https://cloudnativelinz.github.io>.
```

Comment:
I am not familiar with the <http..> syntax and used it without the brackets until now. Could you please let me know on the reasons and advantages of this syntax?

Reply:
Interestingly, I just learned there are multiple ways to display URLs in markdowns: https://meta.stackexchange.com/questions/13501/links-to-urls-containing-parentheses

Reply:
I have a MD plugin that complains about bare-urls.
Here is the explanation of it:
https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md034.md

Reply:
ok, fine for me :) thank you for clarification